src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -15,20 +15,23 @@
 -->
 
 <kd-help-section>
-  <md-input-container class="md-block" md-is-error="ctrl.isNameError()">
+  <md-input-container class="kd-deploy-block" md-is-error="ctrl.isNameError()">
```

Comment:
This class is currently ignored anyway. I think we should stick with md-block. 

![zrzut ekranu z 2016-02-09 13-21-08](https://cloud.githubusercontent.com/assets/2285385/12916085/14c1c53c-cf30-11e5-8df3-d3a2d0734d28.png)

Ps. For margins/paddings etc. use multipliers or divisors of $baseline-grid variable.